src/BaselineOfSpec/BaselineOfSpec.class.st

```diff
@@ -1,38 +1,72 @@
 Class {
 	#name : #BaselineOfSpec,
 	#superclass : #BaselineOf,
+	#instVars : [
+		'link'
+	],
 	#category : #BaselineOfSpec
 }
 
+{ #category : #'pre/post load' }
+BaselineOfSpec >> addHackToLoadSpec [
+	"Metacello pop some UIManager request during the loading of Spec. This cause a problem so we remove the need of the UI manager."
+
+	self shouldInstallHack ifFalse: [ ^ self ].
+	
+	link := MetaLink new
+		metaObject: #receiver;
+		selector: #warnAboutLosingChangesForSpec:;
+		arguments: #(#arguments);
+		control: #instead.
+		
+	(MCVersionLoader >> #warnAboutLosingChangesTo:ifCancel:ifMerge:) ast link: link
+]
+
 { #category : #baseline }
-BaselineOfSpec >> baseline: spec [ 
-    
+BaselineOfSpec >> baseline: spec [
 	<baseline>
 
 	| repository |
-	 
 	repository := self packageRepositoryURL.
-       
-	spec for: #common do: [
-		spec baseline: 'Morphic' with: [spec repository: repository].
-		spec package: 'Spec-Core'.
-		spec package: 'Spec-Inspector'.
-		spec package: 'Spec-Layout'.
-		spec package: 'Spec-MorphicAdapters'.
-		spec package: 'Spec-PolyWidgets'.
-		spec package: 'Spec-Tools'.
-		spec package: 'Spec-StubAdapter'.
-		spec package: 'Spec-Deprecated'.
-	].
+
+	spec for: #common do: [ 
+		spec 
+			preLoadDoIt: #addHackToLoadSpec;
```

Comment:
The hacks in baseline should be removed. They are here to update Spec.

Reply:
Done + https://github.com/pharo-spec/Spec/issues/337